services/forms/user_form.go

```diff
@@ -364,6 +367,19 @@ func (f *AddKeyForm) Validate(req *http.Request, errs binding.Errors) binding.Er
 	return middleware.Validate(errs, ctx.Data, f, ctx.Locale)
 }
 
+// AddSecretForm for adding secrets
+type AddSecretForm struct {
+	Title           string `binding:"Required;MaxSize(50)"`
```

Comment:
I believe it was copied from:
https://github.com/go-gitea/gitea/blob/87c64f655a6085f23499156504e039c13da77b21/services/forms/user_form.go#L350-L358

It may not be a bad thing to be consistent, and I think 50 is quite enough.